lib/libc/common/source/signal/signal.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2025 The Linux Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
@adityachopra29 - I'm not a laywer, but unless you are an employee of LF, it's probably wise to not assign copyright to them.

Recently, Zephyr's [Contribution Guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#copyright-and-license-notices) were updated, so maybe this would work better?

```suggestion
 * SPDX-License-Identifier: Apache-2.0
 * SPDX-FileCopyrightText: Copyright The Zephyr Project Contributors
```